tests/unittest/_torch/executor/test_kv_cache_transceiver_contract.py

```
@@ -0,0 +1,310 @@
+
+def test_results_stay_positionally_compatible() -> None:
+    fake = FakeKvCacheTransceiver()
+    req = _req(9)
+    fake.respond_and_send_async(req)
+    fake.finish_send(req)
+
+    completed, errors = fake.check_context_transfer_status(0)
+    assert (completed, errors) == ([9], [])
+
+    gen = fake.check_gen_transfer_status(0)
+    completed, errors, cancelled = gen
```

Ruff (0.16.2)

[warning] Unpacked variable `cancelled` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)